src/demos/SpinnerDemo.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react'
+import Spinner from '../components/spinner/Spinner'
+import spinnerDocPath from '../docs/spinner.md'
+import ReactMarkdown from 'react-markdown'
+
+const SpinnerDemo=()=>{
+const [markdown,setMarkdown]=useState('')
+
+useEffect(()=>{
+    fetch(spinnerDocPath).then((response)=> response.text()).then((text)=>{
+        setMarkdown(text)
+    })
+
+})
```

Comment:
Why are we not adding anything in the dependencies array? Not providing that in `useEffect` will lead to serious issues of memory leaks and can cause the performance.